### PR TITLE
OP-561 Improve StockCard dialog; add title and space

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-561

Attempted to make the least change yet make the dialog look better.

### CODE REVIEW

**Comment by dbmalkovsky** [comment]
The date from/to dialog is also used by the Stock Ledger option.

File: src/main/java/org/isf/utils/jobjects/JFromDateToDateChooser.java [hunk 1]
```diff
@@ -41,7 +40,7 @@
  */
 public class JFromDateToDateChooser extends JPanel {
 	
-	private static final String DATE_TIME_FORMAT = "dd/MM/yyyy";
+	private static final String DATE_TIME_FORMAT = " dd/MM/yyyy";
```
Comment (mwithi) [inline src/main/java/org/isf/utils/jobjects/JFromDateToDateChooser.java:43]: There was no other layout adjustment other than adding a space in the format?

Comment (mwithi) [reply src/main/java/org/isf/utils/jobjects/JFromDateToDateChooser.java:43]: if I understand correctly the change...

**Review by mwithi — COMMENTED** [review]
Just one question

File: src/main/java/org/isf/utils/jobjects/JFromDateToDateChooser.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/utils/jobjects/JFromDateToDateChooser.java:43]: Yes; I added a space so when it figures out the preferred spacing for the field there is actually enough room to see the whole date.  I was trying to make the most simple change in the code and this is what I came up with :)

Comment (mwithi) [reply src/main/java/org/isf/utils/jobjects/JFromDateToDateChooser.java:43]: what about setting the preferred size?
`dateFromDateChooser.setPreferredSize(new Dimension(90, 20));`

Comment (dbmalkovsky) [reply src/main/java/org/isf/utils/jobjects/JFromDateToDateChooser.java:43]: I was going to do that but at the time I considered that a "bigger" change that you might not approve.  Besides as shown in another PR figuring out the exact size can be tricky.

Comment (mwithi) [reply src/main/java/org/isf/utils/jobjects/JFromDateToDateChooser.java:43]: > as shown in another PR figuring out the exact size can be tricky

It depends by the layout manager involved, some honor preferredSize() others ignore it and eventually require different approaches to obtain the wanted result. For the FlowLayout it is ok to use it, although it should guess the size by the JComponent, that in this case is a compound one, that's why I think it's tricky fo it to guess.

Comment (dbmalkovsky) [reply src/main/java/org/isf/utils/jobjects/JFromDateToDateChooser.java:43]: So what is the final decision?  Leave the code as in this PR or replace it with `setPreferredSize()`?

Comment (mwithi) [reply src/main/java/org/isf/utils/jobjects/JFromDateToDateChooser.java:43]: I think it is safe to use `setPreferredSize()` in this case (although there could be a better way to make the FlowLayout manager to calculate the rendering size by its own)

**Review by mwithi — APPROVED** [review]
Thanks a lot!

**Pull request merged**